### ISSUE TITLE
cuda and cudnn version

### ISSUE THREAD

**Issue by mudong0419** [golbin/TensorFlow-Multi-GPUs#1]
Thanks for your work. What are versions of your cuda and cudnn?

**Comment by golbin** [comment]
I use cuda 8 and cudnn 5, 6 but I think it doesn’t care.

**Comment by ahnHeejune** [comment]
Thanks for your sharing the code.   But, your code does not work on my machine with 3 Tian Xp GPUs.   I spent some time and I found it run with " optimizer = tf.train.AdamOptimizer(learning_rate).minimize(
        loss, colocate_gradients_with_ops=False)"  not with True.  What do you think about this?
